best-practices/content/30_Antivirus.tex

```diff
@@ -0,0 +1,64 @@
+\chapter{\acrlong{av}}
+\section{Allgemeines} %TODO: maybe change title
+Ein \acrfull{av} ist der erste Verteidigungsmechanismus in einem Netzwerk. Angriffe, darunter Schadsoftware, kommen in den meinsten Fällen über einen Endpoint (z.B. Notebook) in ein Firmennetzwerk.\\
+
+Eine gegebene Struktur ist immer nur so stabil, wie sein schwächstes Glied. Dies gilt bei Häusern wie auch bei Computernetzwerken.
+Oftmals ist das schwächste Glied ein Client oder ein Server, da diese Geräte vielfach mit dem Internet kommunizieren.
+Dies macht genau diese Geräte besonders schützenswert.\\
+
+Server \& Clients werden mit einem \acrlong{av} versucht zu schützen. Ein Antivirus hilft bekannte Schadsoftware zu entfernen.
+Neue Schadsoftware wird versucht mit komplexen Algorithmen zu erkennen und deren Vorhaben zu blockieren.
+Schadsoftware kann sich sehr schnell vermehren und sich im ganzen Netzwerk ausbreiten. 
+Das kann den Effekt haben, dass die ganze Unternehmung beeinträchtigt wird.
+Deshalb ist eines der wichtigesten Erkenntnisse, dass ein \acrlong{av} auf \textbf{allen} Endpoints (Server \& Client) benötigt wird.
+
+
+
+
+
+\section{Windows Defender}
+\acrfull{av} kommen in allen Arten \& Formen.
+Es ist schwer den Überblick über die Landschaft der Antiviren Programme zu behalten.\\
+
+
+Microsoft bietet für seine Produkte einen eigenen kostenfreien \acrlong{av} an.
+Der Windows Defender ist weder der beste \acrshort{av} noch hat er die meisten Features.
```

Comment:
Würi jetzt so ned sege:

https://www.microsoft.com/security/blog/2021/05/11/gartner-names-microsoft-a-leader-in-the-2021-endpoint-protection-platforms-magic-quadrant/

Reply:
review again @sevigrimm